src/common/copilot/telemetry/copilotTelemetry.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+import { ITelemetry } from "../../../client/telemetry/ITelemetry";
+
+// Telemetry Event Names
+export const CopyCodeToClipboardEvent = 'CopyCodeToClipboardEvent';
```

Comment:
Move all the event names in some constant.

Reply:
done